recipes/pypolyagamma-3/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+if [ $(uname) == Darwin ]
+then
+    export CC=gcc CXX=g++
+fi
+
+if [ $(uname) == Linux ]
+then
+    yum install gsl-devel
```

Comment:
I tried using conda forge's gsl. I get an error when the code hits `#include <gsl/gsl_rng.h>`. I ran into the same thing when trying to configure a .travis.yml the other day. The way I fixed it there was to depend on libgsl0-dev but I'm not sure I to do that here.

As for all the other comments being left, thanks, I'm new to CI, conda, conda forge, ... etc. So the comments are very helpful to me.